package.json

```diff
@@ -36,21 +36,14 @@
     "extract_abi": "./scripts/extract_abi.js",
     "truffle-test": "yarn tsc && npx truffle test",
     "truffle-compile": "truffle compile",
-    "webtools-pack": "webpack-cli --config ./webtools.webpack.js",
```

Comment:
how can this run? we have a test to pack the jsrelay.

Reply:
this is not jsrelay, this is webtools